### PR TITLE
Fix: Remove "anta" from requirements while the integration is in preview

### PR DESCRIPTION
**Pull request by ClausHolbechArista**
Remove "anta" from requirements while the integration is in preview

### CODE REVIEW

File: .github/workflows/pull-request-management.yml [hunk 1]
```diff
@@ -343,10 +343,12 @@ jobs:
         avd_scenario:
           - 'eos_validate_state'
         ansible_version:
-          - 'ansible-core<2.16.0 --upgrade'
+          # Temporarily installing anta manually while the feature is in preview.
+          - 'ansible-core<2.16.0 anta==0.9.0 --upgrade'
```
Comment (gmuloc) [inline .github/workflows/pull-request-management.yml]: can we make a file for this so we can change the value in only one place instead?

**Review by gmuloc — COMMENTED** [review]
Missing something in eos_validate_state task for verify requirements to check for anta

**Review by carl-baillargeon — COMMENTED** [review]
Please update to 0.10.0. Thanks!

**Review by gmuloc — APPROVED** [review]

**Review by carl-baillargeon — APPROVED** [review]
LGTM

**Pull request merged**